src/kernel/src/loader/mod.rs

```diff
@@ -245,9 +245,10 @@ pub fn load_cmdline(
     }
 
     let end = guest_addr
```

Comment:
Can we rename this variable from `end` to `last` also in order to make this code more readable ?

Reply:
Yep. Done.